### ISSUE TITLE
Create Card component.

### ISSUE THREAD

**Issue by iPzard** [default-services/components#6]
Create a card component, it should utilize the custom properties for borders and box shadows - see: custom-properties.scss.

**Comment by iPzard** [comment]
@kayroy247 Feel free to remove yourself from this issue and switch to another, if you prefer.

**Comment by kayroy247** [comment]
@iPzard Looks good. I will check up the link to the reference on custom properties and work on the card component this weekend. :smiley:

**Comment by iPzard** [comment]
Thanks @kayroy247 😃! Let me know if any questions come up, happy to help.

**Comment by iPzard** [comment]
@kayroy247 I took your advice about having some sort of contributing docs, these should help people find there way around the project a little easier, and keeping things consistent: [Contributing to Default](https://github.com/default-services/components/wiki/Contributing-to-Default)

**Comment by kayroy247** [comment]
Yes, @iPzard I saw the contributing docs on your comment on Facebook and I have also saved it so as to follow a consistent convention. I will reach out to you if I have any questions. 😃 👍 💯

**Comment by iPzard** [comment]
Awesome, hope it helps! 🙂
